src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORM = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom.trim(), DATE_FORM);
+        LocalDate toDate = LocalDate.parse(dateTo.trim(), DATE_FORM);
+
+        StringBuilder report = new StringBuilder();
+        report.append("Report for period ")
+                .append(dateFrom)
+                .append(" - ")
+                .append(dateTo)
```

Comment:
As per the checklist, it's better to use `fromDate` and `toDate` (which are already formatted as `LocalDate` objects) instead of `dateFrom` and `dateTo` when appending to the `report`. This ensures that the dates are consistently formatted using the `DATE_FORM` formatter.